examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,619 @@
+   "source": [
+    "The next on-demand simulator is ***DM1***. This is a density matrix simulator which, like SV1, allows you to simulate a larger number of qubits, as well as take advantage of batch execution. \n",
+    "\n",
+    "**Recommended use case:** Noisy variational algorithms on up to 17 qubits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "54e855a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aws_dm1 = provider.get_backend(\"dm1\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae97b1a1",
+   "metadata": {},
+   "source": [
+    "Lastly for on-demand simulators, we have ***TN1***. This is a tensor-network simulator, which represents each gate in a circuit as a tensor. TN1 can simulate a larger number of qubits for circuits with local gates or other special structure, but will typically be slower than SV1 or DM1 for circuits with long-range or all-to-all gate structure.\n",
+    "\n",
+    "**Recommended use case:** Noisless quantum circuits with local connectivity and up to 50 qubits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "cf7e09dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aws_sv1 = provider.get_backend(\"TN1\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad54a536",
+   "metadata": {},
+   "source": [
+    "**Note**: Each AWS resource, (like a file or CPU or QPU) lives in a specific region and may only be accessible from that region. One of the managed simulators (TN1) is only available in the `eu-west-2`, `us-east-1`, and `us-west-2` regions. \n",
+    "\n",
+    "To change your AWS region if you're running in a managed notebook, you'll need to use the GUI in the top righthand corner of the AWS console to select your new region, then relaunch or create your notebook from the Braket console.\n",
+    "\n",
+    "To change your AWS region if you're running in your local development environment, you run the following code snippet:\n",
+    "```\n",
+    "import os\n",
+    "os.environ[\"AWS_REGION\"] = \"your-desired-region\"\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc6910f0",
+   "metadata": {},
+   "source": [
+    "### Quantum Processing Units (QPUs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bfc8706",
+   "metadata": {},
+   "source": [
+    "Amazon Braket also provides acess to a number of third-party quantum hardware devices. The following code shows how to view the supported QPUs which are currently online:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "f5329bca",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[BraketBackend[Aspen-M-3], BraketBackend[Lucy]]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "provider.backends(statuses=[\"ONLINE\"], types=[\"QPU\"])"
```

Comment:
Do we run the risk of a customer running this on a different device than m-3?

Reply:
> Changed to using M3.